src/Queue/QueueHandler.php

```diff
@@ -2,47 +2,53 @@
 
 namespace CacheWerk\BrefLaravelBridge\Queue;
 
-use Throwable;
 use RuntimeException;
 
+use Aws\Sqs\SqsClient;
+
 use Bref\Context\Context;
 use Bref\Event\Sqs\SqsEvent;
 use Bref\Event\Sqs\SqsHandler;
 
+use Illuminate\Queue\SqsQueue;
+use Illuminate\Queue\Jobs\SqsJob;
+use Illuminate\Queue\QueueManager;
+use Illuminate\Queue\WorkerOptions;
+
 use Illuminate\Container\Container;
 use Illuminate\Contracts\Events\Dispatcher;
 use Illuminate\Contracts\Debug\ExceptionHandler;
 
-use Illuminate\Log\LogManager;
-
-use Illuminate\Queue\SqsQueue;
-use Illuminate\Queue\Jobs\SqsJob;
-use Illuminate\Queue\QueueManager;
-use Illuminate\Queue\Events\JobProcessed;
-use Illuminate\Queue\Events\JobProcessing;
-use Illuminate\Queue\Events\JobExceptionOccurred;
+use CacheWerk\BrefLaravelBridge\MaintenanceMode;
 
 class QueueHandler extends SqsHandler
```

Comment:
Vapor uses the `VAPOR_QUEUE_DATABASE_SESSION_PERSIST` option to keep database connections open between SQS events, should we add this as well?

Reply:
https://github.com/laravel/vapor-core/blob/45efd886ba196e4acb7a218e78e4eb8183358a48/src/Runtime/Handlers/QueueHandler.php#L88-L93

Reply:
@tillkruss Without a doubt!

Reply:
@JackEllis This looks good to me now. Any feedback?

@georgeboot pointed out in Slack:

> Currently there are no database disconnects, which means connections always stay open between invocations. AFAICT this is the case for web, artisan and queue. Only the octane client optionally disconnects.